### PR TITLE
fix: add devexp-sdks for api changes

### PR DESCRIPTION
**Pull request by wtrocki**
## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,3 +13,7 @@ internal/connector/**/*  @valdar @dhirajsb
 # common
 pkg/**/*  @pb82 @davidffrench @machi1990 @akoserwal @pawelpaszki @JameelB @miguelsorianod @ziccardi @valdar @dhirajsb
 test/**/* @pb82 @davidffrench @machi1990 @akoserwal @pawelpaszki @JameelB @miguelsorianod @ziccardi @valdar @dhirajsb
+
+# api
+openapi/ @bf2fc6cc711aee1a0c2a/devexp-sdks
```
Comment (machi1990) [inline .github/CODEOWNERS:18]: Github is complaining with `Unknown owner on line 18: make sure the team @bf2fc6cc711aee1a0c2a/devexp-sdks exists and has write access to the repository`
is that to be expected?

**Review by machi1990 — APPROVED** [review]
lgtm

I've left a comment on the added owner

**Comment by wtrocki** [comment]
Strange. team exists: https://github.com/orgs/bf2fc6cc711aee1a0c2a/teams/devexp-sdks Syntax seems to be ok

**Comment by wtrocki** [comment]
Ok I know what is the problem:

Fix: https://github.com/bf2fc6cc711aee1a0c2a/org-management/pull/333

**Comment by wtrocki** [comment]
I guess we can merge it assuming that other PR will be merged as well

**Comment by machi1990** [comment]
Thanks @wtrocki merged.

**Pull request merged**